backend/dataall/modules/mlstudio/api/resolvers.py

```diff
@@ -0,0 +1,137 @@
+import logging
+from dataall.api.context import Context
+from dataall.db import exceptions
+from dataall.api.Objects.Stack import stack_helper
+from dataall.modules.mlstudio.api.enums import SagemakerStudioRole
+from dataall.modules.mlstudio.services.services import SagemakerStudioService, SagemakerStudioCreationRequest
+from dataall.modules.mlstudio.db.models import SagemakerStudioUser
+
+log = logging.getLogger(__name__)
+
+
+class RequestValidator:
```

Comment:
Should we generalize this class and reuse it in the other modules as well? I can see a similar one existing for notebooks.
Probably the required parameters (contained in a request) could mainly differ among the different modules

Reply:
It's not a must-have. I think for validation of requests with big payloads (like creation of an entity) it can be reasonable. But for small requests it's very optional.